packages/fast-components-styles-msft/src/breadcrumb/index.ts

```diff
@@ -0,0 +1,45 @@
+import designSystemDefaults, {
+    DesignSystem,
+    withDesignSystemDefaults,
+} from "../design-system";
+import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
+import { applyLocalizedProperty, Direction } from "@microsoft/fast-jss-utilities";
+import { ensureForegroundNormal } from "../utilities/colors";
+import { BreadcrumbClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { applyTypeRampConfig } from "../utilities/typography";
+import { fontWeight } from "../utilities/fonts";
+
+const styles: ComponentStyles<BreadcrumbClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<BreadcrumbClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const direction: Direction = designSystem.direction;
+
+    return {
+        breadcrumb: {
+            padding: "4px",
+        },
+        breadcrumb_item: {
+            fontWeight: `${fontWeight.bold}`,
+            display: "inline",
+            ...applyTypeRampConfig("t7"),
+        },
+        breadcrumb_item__current: {},

```

Comment:
What happens if a hypertext is passed with an href value? I would assume to ensure consistency we would want to use this within the MSFT package. Otherwise, what purpose is it serving if we expect that users style their current item to meet our requirements?